src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/get_system_capability_request.cc

```diff
@@ -128,10 +131,70 @@ void GetSystemCapabilityRequest::Run() {
         return;
       }
       break;
+    case mobile_apis::SystemCapabilityType::APP_SERVICES: {
+      smart_objects::SmartObject app_service_capabilities(
+          smart_objects::SmartType_Map);
+      smart_objects::SmartObject supported_types(
+          smart_objects::SmartType_Array);
+      smart_objects::SmartObject app_services(smart_objects::SmartType_Array);
+
+      std::vector<smart_objects::SmartObject> service_records =
+          application_manager_.GetAppServiceManager().GetAllServices();
+      std::set<mobile_apis::AppServiceType::eType> service_types;
+
+      for (auto& record : service_records) {
+        // SUPPORTED TYPES
+        mobile_apis::AppServiceType::eType service_type =
+            static_cast<mobile_apis::AppServiceType::eType>(
+                record[strings::service_manifest][strings::service_type]
+                    .asUInt());
+        service_types.insert(service_type);
+
+        // APP SERVICES
+        smart_objects::SmartObject app_services_capabilities(
+            smart_objects::SmartType_Map);
+        app_services_capabilities[strings::updated_app_service_record] = record;
```

Comment:
I think this is supposed to only include active service types. It looks like this could add multiple of the same service type.

Reply:
GetAllServiceRecords does not do a check for if the service is active.

Reply:
At the moment it adds all published services to the list of service records. Not sure if that is what we wanted for the OSCU notification and we can modify the behavior if required. 

Even though it will list multiple services of the same service type the service record contains a `serviceActive` param